styles.css

```diff
@@ -0,0 +1,145 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap");
+* {
+    font-family: "Poppins", sans-serif;
```

Comment:
Add font as a root variable and use the root variable instead reset-CSS font-family globally